docs/REFERENCE.md

```diff
@@ -210,6 +210,13 @@ index = <index>
     * ONLY VALID WITH outputMode SPLUNKSTREAM
     * Splunk index to write events to. Defaults to main if none specified.
     
+extendIndexes = <index_prefix>:<num>,<index2>,<index3>
+    * Sample level setting. Use this setting enable eventgen to generate multi indexes for one sample. 
+    * if you set the value with pattern like "<index_prefix>:<num>", it will treat <index_prefix> as a prefix of an actual index, 
+    * and <num> is the count of index you want to extend for this sample.
```

Comment:
this num is actually like a weight. Can we change the wording as a weight?

Reply:
Yes, it's better to describe it using the word "weight".